packages/react-native-win32/transformer-selector.js

```diff
@@ -0,0 +1,11 @@
+// For React Native version 0.59 or later
```

Comment:
1. We need a copyright header
2. What's with the comment? This repo is kind of the living source of truth for version. Was this copied from somewhere?

Reply:
This is from react-native-svg-transformer.  It actually has its own functionality to switch between 2 transformers: the default metro one and its own svg transformation. In the case of a simple project where you only need the metro transformer and svg support, you can simply set it to be the metro transformer and it will do switching between these two. However I thought it would be pretty weird for a project like ours to set the metro transformer to something labelled an svg transformer so I refactored the generic transform selector code into its own module where we can just drop in new transformers and file extensions.